### DISCUSSION TITLE
Supporting more hardware / devices - Possible list of candidates

### DISCUSSION THREAD

**Discussion by j1nx** [OpenVoiceOS/ovos-buildroot#55] [Ideas]
## Information
Initial development of MycroftOS is being done on the Raspberry Pi 3B as that board is currently present within the Mycroft A.I. Mark-I. However that is not the only suitable piece of hardware that could run MycroftOS and therefor the Mycroft .A.I. software stack. To add to that, other hardware devices might even be more suitable for this type of purpose...

This "Issue" is to identify the different type of hardware devices currently available or available within the near future that are of interest to be supported by MycroftOS.

### Raspberry Pi Foundation
- Raspberry Pi 3B (Currently supported as initial development device)
- Raspberry Pi 3B+ (Buildroot-wise the same as the 3B, so most likely supported already)
- Raspberry Pi 3 / 2 / CM / zero / zero W (These boards might not be powerfull enough according to Mycroft A.I. However as Buildroot is such a minimalistic OS a test would be needed to confirm. You never know)

### Hardkernel
- Odroid-C2 (Amlogic S905 support)
- ODROID-XU4 (Exynos5 Octa support)

### Asus
- Tinkerboard (Rockchip RK3288 support)

### FriendlyARM
- NanoPi M4

### Rockchip boards (unknown supplier for now)
- RK3399
- RK3399Pro (includes a NPU)

### UDOO
- UDOO X86 ULTRA (or basically X86 support)

... Issue open for discussion and / or suggestions ...

**Comment by Olzeke51** [comment]
1) Is the Pi4 a candidate?
2) Is the MycroftAI Mark2 [dev-kit and/or future production unit] a candidate?

**Reply by ChanceNCounter** [reply]
These are the only two targets at present. This ticket is stale. @j1nx should we edit this, or just close it?

**Comment by lipschitzallee** [comment]
The CutiePi Tablet would be great:
https://github.com/cutiepi-io/cutiepi-doc

**Reply by builderjer** [reply]
Like on top of their software? Or just support for the device?  I don't see why you couldn't put the buildroot image on there as an os. There may be a couple of thinks that don't work, maybe the multiple cameras, and the gyro probably. But with some testing, a phal plugin could probably br made for that

**Comment by j1nx** [comment]
@j1nx Ping for self. Really need to update this one.
